cdap-ui/app/cdap/components/Experiments/CreateView/SplitDataStep/SplitInfoTable/index.js

```diff
@@ -0,0 +1,345 @@
+              >
+                <td>{field.name}</td>
+                <td>{field.numTotal}</td>
+                <td>{field.numEmpty}</td>
+                <td>{field.numZero}</td>
+                <td>{field.mean}</td>
+                <td>{field.stddev}</td>
+                <td>{field.min}</td>
+                <td>{field.max}</td>
+              </tr>
+            );
+          })
+        }
+      </tbody>
+    );
+  };
+
+  renderCategoricalTableBody = (fields) => {
+    return (
+      <tbody>
+        {
+          fields.map(field => {
+            return (
+              <tr
+                key={field.name}
+                onClick={this.props.onActiveColumnChange.bind(null, field.name)}
+              >
+                <td>{field.name}</td>
+                <td>{field.numTotal}</td>
+                <td>{field.numEmpty}</td>
+                <td>{field.unique}</td>
+              </tr>
+            );
+          })
+        }
+      </tbody>
+    );
+  };
+
+  renderNumericalTable = (fields) => {
+    return (
+      <SortableTable
+        entities={fields}
+        tableHeaders={this.NUMERICAL_FIELD_HEADERS}
+        renderTableBody={this.renderNumericalTableBody}
+      />
+    );
+  };
+
+  renderCategoricalTable = (fields) => {
+    return (
+      <SortableTable
+        entities={fields}
+        tableHeaders={this.CATEGORICAL_FIELD_HEADERS}
+        renderTableBody={this.renderCategoricalTableBody}
+      />
+    );
+  };
+
+  renderTable = () => {
+    if (!(typeof this.state.splitInfo.schema === 'object' && this.state.splitInfo.schema.fields)) {
+      return null;
+    }
+    const schema = this.state.splitInfo.schema;
+    const getFieldType = (field) => {
+      let type;
+      // TODO: The assumption fields cannot have complex type coming from dataprep wrangler
+      // Need to verify this.
+      if (Array.isArray(field.type)) {
+        type = field.type.filter(t => t !== 'null').pop();
+      } else if (typeof field.type === 'string') {
+        type = field.type;
+      }
+      return type;
+    };
+    const getStats = ({name: fieldName}) => {
+      const stat = this.state.splitInfo.stats
+        .filter(stat => stat.field === fieldName)
+        .map(stat => ({
+          numTotal: objectQuery(stat, 'numTotal', 'total') || '--',
+          numNull: objectQuery(stat, 'numNull', 'total') || '--',
+          numEmpty: objectQuery(stat, 'numEmpty', 'total') || '--',
+          unique: objectQuery(stat, 'unique', 'total') || '--',
+          numZero: objectQuery(stat, 'numZero', 'total') || '--',
+          numPositive: objectQuery(stat, 'numPositive', 'total') || '--',
+          numNegative: objectQuery(stat, 'numNegative', 'total') || '--',
+          min: objectQuery(stat, 'min', 'total') || '--',
+          max: objectQuery(stat, 'max', 'total') || '--',
+          stddev: objectQuery(stat, 'stddev', 'total') || '--',
+          mean: objectQuery(stat, 'mean', 'total') || '--'
+        }))
+      .pop();
```

Comment:
if what you need is just the last element, wouldn't it be better to do `pop` first and then do the mapping of the values? Then you are not wasting on operation for the other items. Or perhaps use lodash `findLast` method?